app/src/main/java/com/taf/shuvayatra/ui/fragment/HomeFragment.java

```diff
@@ -94,15 +94,18 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
 
         if (showCountryWidget) {
             try {
-                String countryName = selectedCountry.split(",")[Country.INDEX_TITLE_EN].substring(0, 1).toUpperCase() +
+                String countryName = selectedCountry.split(",")[Country.INDEX_TITLE];
+                String countryNameEn = selectedCountry.split(",")[Country.INDEX_TITLE].substring(0, 1).toUpperCase() +
                         selectedCountry.split(",")[Country.INDEX_TITLE_EN].substring(1,
                                 selectedCountry.split(",")[Country.INDEX_TITLE_EN].length());
                 Logger.e(TAG, "selectedCountry.split(): " + Arrays.toString(selectedCountry.split(",")));
-                mCountryWidget = new CountryWidgetModel(countryName);
+                mCountryWidget = new CountryWidgetModel(countryName, countryNameEn);
+                mCountryWidget.setId(Long.parseLong(selectedCountry.split(",")[0]));
             } catch (ArrayIndexOutOfBoundsException e) {
                 Logger.e(TAG, ">>> country name: " + selectedCountry);
                 String countryName = selectedCountry.split(",")[1];
-                mCountryWidget = new CountryWidgetModel(countryName);
+                String countryNameNp = selectedCountry.split(",")[0];
```

Comment:
@ngima are you sure this returns the country name np?? isn't the 0 index supposed to represent country id?